lib/Db/PersonMapper.php

```diff
@@ -49,6 +48,15 @@ public function find(string $userId, int $personId): Person {
 		return $person;
 	}
 
+	public function findByName(string $userId, string $personName): array {
```

Comment:
consider exercising this method in `MergeClusterToDatabaseTest` instead of some of those `findAll` methods (should be relatively easy to just change `findAll` with N `findByName`

Reply:
I'm still trying to fix travis .. I don't want to change more things .. :disappointed: